### ISSUE TITLE
Support WPF?

### ISSUE THREAD

**Issue by Lorymi** [microsoft/winrtc#82]
Does this project support WPF?

**Comment by aisha-magsi11** [comment]
No, we don't have an out-of-box WinRTC solution for WPF or WinForm developers yet, however we'd love to learn about how you would use WinRTC so that we can include that in our product direction as we build for WPF/WinForm.

**Comment by jonlindeheim** [comment]
There are alot of existing WPF applications out there that could benefit from using WinRTC. We are using WebRTC in web based Video Management Systems and would love to be able to use the same technology in our existing WPF apps.

**Comment by Lorymi** [comment]
@aisha-magsi Thanks for reply.
I need to implement video calling in my WPF version for my messenger.
In Android (JAVA) and iOS (SWIFT) there are many WebRTC SDK and library to use
But WPF???
@jonlindeheim Thanks.

**Comment by aisha-magsi11** [comment]
Thanks @jonlindeheim and @Lorymi, that's exactly what we're looking to hear! We believe there is an untapped market with WPF and WinForm developers and we are actively investigating this further so we can bring WinRTC solutions to more native Windows apps very soon. Our upcoming Ignite conference will allow us to meet with more WPF/WinForm developers and hold a focus group session. If you are interested in joining us there, details for the conference and focus group will be posted on our GitHub shortly.

**Comment by Lorymi** [comment]
@aisha-magsi Thanks, this is good news.

**Comment by jonlindeheim** [comment]
@aisha-magsi Any news on the WPF support? Where there any discussions during Ignite?

**Comment by bitbound** [comment]
FYI, I'm using https://github.com/microsoft/MixedReality-WebRTC in desktop apps (console and WPF).  It's been working perfectly.

Not sure how these two projects will overlap in the end. It seems like Microsoft should be combining resources, though.

**Comment by Avidon0** [comment]
@lucent-sea unfortunately, MixedReality-WebRTC seems to be lacking H264 support for WPF

**Comment by bitbound** [comment]
@Avidon0 Oh, I see.  Nevertheless, it's incredibly exciting to see Microsoft working on WebRTC for .NET!  I've been hoping for this for a long time.
